### PR TITLE
sql-statements:add drop-stats.md

### PR DESCRIPTION
**Pull request by SunRunAway**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3337
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-drop-stats.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: DROP STATS
+summary: An overview of the usage of DROP STATS for the TiDB database.
+category: reference
+---
+
+# DROP STATS
+
+`DROP STATS` 语句用于从当前所选定的数据库中删除选定表的统计信息。
```
Comment (yikeke) [inline sql-statements/sql-statement-drop-stats.md:9]:
```suggestion
`DROP STATS` 语句用于从当前所选定的数据库中删除选定表的统计信息。
```
中文？😂 @SunRunAway

Comment (SunRunAway) [reply sql-statements/sql-statement-drop-stats.md]: 忘记push了， PTAL

File: sql-statements/sql-statement-drop-stats.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+title: DROP STATS
+summary: An overview of the usage of DROP STATS for the TiDB database.
+category: reference
+---
+
+# DROP STATS
+
+`DROP STATS` statement is used to delete the statistical information of the selected table from the selected database.
```
Comment (yikeke) [inline sql-statements/sql-statement-drop-stats.md:9]:
```suggestion
The `DROP STATS` statement is used to delete the statistics of the selected table from the selected database.
```

**Pull request merged**